kernel/packages/unity-interface/dcl.ts

```diff
@@ -138,9 +125,10 @@ export async function initializeEngine(_gameInstance: GameInstance) {
 
   if (!EDITOR) {
     await startGlobalScene(unityInterface, 'dcl-gs-avatars', 'Avatars', hudWorkerUrl)
+
     // Temporal: Try to create several global scenes
-    await startPortableExperienceScene(unityInterface, 'dcl-pe-example1', 'pe1')
-    await startPortableExperienceScene(unityInterface, 'dcl-pe-example2', 'pe2')
+    await spawnPortableExperienceScene('pe1')
+    await spawnPortableExperienceScene('pe2')
```

Comment:
this should be removed before merge

Reply:
Sure, it'll be done in https://github.com/decentraland/explorer/pull/1959